### PR TITLE
Cloud hosted README adjustments

### PR DESCRIPTION
**Pull request by AustinSeto**
Small formatting fix, the `+` for hard line breaks in `.adoc` appears as a literal `+` sign on the Cloud Hosted version of the guides.

### CODE REVIEW

**Review by gkwan-ibm — CHANGES_REQUESTED** [review]
@AustinSeto would you also fix the following:
- **Building and running the application**
  - [x] after startContainer, the text "... localhost:9080/query/systemLoad..." -> " http:localhost:9080/query/systemLoad..." 
  - [x] change the "The services take some time..." to "The services may take several minutes to become available." for cloud-hosted guide only
- **Testing the query microservice** 
  - [ ] add an extra statement to tell users "Tests may take several minutes to complete."

File: README.adoc [hunk 1]
```diff
@@ -208,13 +209,13 @@ include::{common-includes}/os-tabs.adoc[]
 
 // static guide instructions:
 ifndef::cloud-hosted[]
-The services take some time to become available. 
-You can access the application by making requests to the `query/systemLoad` endpoint by going to http://localhost:9080/query/systemLoad[localhost:9080/query/systemLoad^].
+The services take some time to become available.  
+You can access the application by making requests to the `query/systemLoad` endpoint by going to http://localhost:9080/query/systemLoad[http://localhost:9080/query/systemLoad^].
```
Comment (dmuelle) [inline README.adoc:213]: You can access the application by making requests to the `query/systemLoad` endpoint by going to the http://localhost:9080/query/systemLoad[http://localhost:9080/query/systemLoad^] URL.

File: README.adoc [hunk 2]
```diff
@@ -37,7 +37,8 @@ You will learn how to build a MicroProfile Rest Client to access remote RESTful
 You'll update the template interface for a MicroProfile Rest Client, which maps to the remote service that you want to call, to use the `CompletionStage` return type.
```
Comment (dmuelle) [inline README.adoc:37]: suggested edit:

You'll update the template interface for a MicroProfile Rest Client to use the `CompletionStage` return type. 

The template interface maps to the remote service that you want to call.

**Review by dmuelle — COMMENTED** [review]
a few suggestions

**Comment by AustinSeto** [comment]
Thank you @dmuelle, all suggestions implemented

**Review by dmuelle — APPROVED** [review]
LGTM

**Review by gkwan-ibm — APPROVED** [review]

**Pull request merged**